app/models/box.rb

```diff
@@ -22,6 +22,7 @@ class Box < ApplicationRecord
 
   has_many :message_threads, extend: MessageThreadsExtensions, dependent: :destroy
   has_many :messages, through: :message_threads
+  has_many :message_submission_requests, dependent: :destroy
```

Comment:
Rozmyslam ci toto nie je ina domena.

Reply:
Nejaka napr. Billing domena myslis? @jsuchal

Reply:
Ci chceme rozlisovat napr. UPVS odoslane spravy, FS odoslane spravy,...?

Reply:
> Billing domena

Nie nie, skor uvazujem tak, ze v GOPRO neexistuje billing domena. Tato domena existuje niekde v CRM nad tym, ktory si vytiahne co potrebuje z nejakej "statistics" domeny.

Reply:
> rozlisovanie
podla mna nie

Reply:
Ako to teda spravime @jsuchal? Chceme to iba presunut do statistics domeny?